src/diffusers/pipelines/stable_diffusion_xl/pipeline_flax_stable_diffusion_xl.py

```diff
@@ -253,7 +254,7 @@ def loop_body(step, args):
             noise_pred = noise_pred_uncond + guidance_scale * (noise_prediction_text - noise_pred_uncond)
 
             # compute the previous noisy sample x_t -> x_t-1
-            latents, scheduler_state = self.scheduler.step(scheduler_state, noise_pred, t, latents).to_tuple()
+            latents, scheduler_state = self.scheduler.step(scheduler_state, noise_pred, t, latents, prng_seed).to_tuple()
```

Comment:
Another option is to create the key inside the scheduler if it's not passed, like FlaxDDPMScheduler does

Reply:
See https://github.com/entrpn/diffusers/pull/1 for a working denoising loop.